cores/esp8266/umm_malloc/umm_malloc.cpp

```diff
@@ -498,9 +498,30 @@
 #include "umm_malloc.h"
 
 #include "umm_malloc_cfg.h"   /* user-dependent */
+#if defined(DEBUG_ESP_PORT)
+#include <assert.h>
+#endif
 
 extern "C" {
 
+#ifdef UMM_CRITICAL_PERIOD_ANALYZE
+static struct _UMM_TIME_STATS time_stats = {
+  {0xFFFFFFFF, 0U, 0U, 0U},
+  {0xFFFFFFFF, 0U, 0U, 0U},
+  {0xFFFFFFFF, 0U, 0U, 0U},
+  {0xFFFFFFFF, 0U, 0U, 0U} };
+
+bool ICACHE_FLASH_ATTR get_umm_get_perf_data(struct _UMM_TIME_STATS *p, size_t size) {
```

Comment:
Suggest moving this function to another .cpp, as previously explained.

Reply:
okay